docs/src/components/componentList/drawerNavItemComponent/BasicDrawerNavItem.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { CodeBlock } from '../../codeSnippet/codeBlock';
+import { Drawer, DrawerBody, DrawerNavGroup, DrawerNavItem } from '@brightlayer-ui/react-components';
+import * as colors from '@brightlayer-ui/colors';
+
+const htmlCode = `<DrawerBody>
+  <DrawerNavGroup>
+    <DrawerNavItem title="Dashboard" icon={<Dashboard/>} itemID="1" />
+    <DrawerNavItem title="Locations" icon={<Notifications />} itemID="2" />
+    <DrawerNavItem title="Legal" icon={<Gavel />} itemID="3" />
+  </DrawerNavGroup>
+</DrawerBody>
+`;
```

Comment:
The rendered code doesn't match the copy/paste code. Notably the styles and the icons.

Reply:
done please check